website/static/js/pages/base-page.js

```diff
@@ -4,15 +4,20 @@
  */
 'use strict';
 // CSS used on every page
-require('../../vendor/bower_components/bootstrap/dist/css/bootstrap-theme.css');
 require('../../vendor/bootstrap-editable-custom/css/bootstrap-editable.css');
 require('../../vendor/bower_components/jquery-ui/themes/base/minified/jquery.ui.resizable.min.css');
 require('../../css/bootstrap-xl.css');
 require('../../css/animate.css');
-require('../../css/site.css');
-require('../../css/navbar.css');
+require('../../../../node_modules/osf-style/css/base.css');
+require('../../css/search-bar.css');
 require('font-awesome-webpack');
 
+// for public activity page under explore nav-tag
+require('../../css/pages/public-activity-page.css');
```

Comment:
Follow my previous recommendation: load critical CSS with `link` tags; load non-critical CSS in page modules. 

I would think these modules should get loaded with link tags. If not, then you can go ahead and create JS page modules for Public Activity and Getting Started.